Packages/com.unity.inputsystem/InputSystem/API/Input.cs

```diff
@@ -937,6 +1006,46 @@ public static bool IsGamepadButtonUp(GamepadButton input, GamepadSlot gamepadSlo
         {
             throw new NotImplementedException();
         }
+#endif
+        
+        private static GamepadSlot FindSlotForGamepad(Gamepad gamepad)
+        {
+            // TODO implement me
+            return GamepadSlot.Slot1;
+        }
+
+        private static float GetGamepadTriggerPressPoint(Gamepad gamepad)
+        {
+            // TODO remove me when init is implemented
+            InitializeGamepadConfigs();
+
+            var gamepadSlot = FindSlotForGamepad(gamepad);
+            switch (gamepadSlot)
+            {
+                case GamepadSlot.Slot1:
+                case GamepadSlot.Slot2:
+                case GamepadSlot.Slot3:
+                case GamepadSlot.Slot4:
+                case GamepadSlot.Slot5:
+                case GamepadSlot.Slot6:
+                case GamepadSlot.Slot7:
+                case GamepadSlot.Slot8:
+                case GamepadSlot.Slot9:
+                case GamepadSlot.Slot10:
+                case GamepadSlot.Slot11:
+                case GamepadSlot.Slot12:
+                    return s_GamepadConfigs[(int)gamepadSlot].triggerPresspoint;
+                case GamepadSlot.All:
+                    return s_GamepadConfigs[0].triggerPresspoint; // TODO what should it mean?
```

Comment:
Agree, similar comment as previously, aggregating this one e.g. min/max or similar would yield a different result than applying it on each individual contributor.

Reply:
Fixed similar to previous method - we throw an exception if all is passed